load-test-framework/src/main/java/com/google/pubsub/clients/mapped/CPSPublisherTask.java

```diff
@@ -25,7 +24,7 @@
   private final String topic;
   private final String payload;
   private final int batchSize;
-  private final PubsubProducer<String, String> publisher;
+  private final KafkaProducer<Double, String> publisher;
```

Comment:
It's not of any importance, I will change the task completely anyway.
I just copied the initializations from another class.